### ISSUE TITLE
Option to change between old and reworked charter

### ISSUE THREAD

**Issue by GoldGal64** [KadeArchive/Kade-Engine#1738]
Using the reworked charter is good, but it may be annoying to use for users who are used to chart with the old charter. Some features such as the usual 16 steps and the section viewer with only vision for a section could be more comfortable for these users.

**Comment by isakube** [comment]
yeah the reworked charter is fine as long as you turn off the checkerboard
